cogs/minigame/minigame.py

```diff
@@ -44,7 +44,9 @@ async def destroy(self, ctx: commands.Context, show_scoreboard: bool = False):
                     if player.id == pair[1][0]:
                         mapped_scores.append('{}) {} with {} points'.format(pair[0] + 1, player.mention, pair[1][1]))
                         break
-            embed = discord.Embed(color=discord.Color.from_rgb(255, 147, 206), title='Minigame ended!', description='Total points:\n{}'.format('\n'.join(mapped_scores)))
+            embed = discord.Embed(color=discord.Color.from_rgb(255, 147, 206), 
```

Comment:
Hex colors look cleaner

```suggestion
            embed = discord.Embed(color=0xFF93CE, 
```